app/Models/Recruiter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Recruiter extends Model
```

Comment:
Do you think that the recruiter is another thing different than a user... or is a user of type recruiter?